README.md

```diff
@@ -7,13 +7,24 @@ Pulumi's framework for Amazon Web Services (AWS) infrastructure.
 This package is meant for use with the Pulumi CLI.  Please visit [pulumi.io](https://pulumi.io) for
```

Comment:
We probably want to call out that this library is currently available for JavaScript/TypeScript (Although I think we also have a partial implementation of Network for Python, which is likely now deprecated?  We'll want to explain this briefly somehow.)